pyiron_base/job/generic.py

```diff
@@ -1629,6 +1629,12 @@ def _reload_update_master(self, project, master_id):
                 "busy master: {} {}".format(master_id, self.get_job_id())
             )
             del self
+            
+    def __enter__(self):
```

Comment:
Since `interactive_open` is now required I find this error message redundant.

Reply:
@samwaseda I agree - removed it.